.github/workflows/.spellcheck-conf.toml

```diff
@@ -0,0 +1,2 @@
+[default]
+extend-ignore-words-re = ["ASSERTne", "ASSER", "Tne"]
```

Comment:
isn't `ASSERTne` enoguh?

Reply:
No, in fact `ASSERTne` is not needed here. That's because the Action splits this into two words from a dictionary: `ASSERT` and `The`. I've removed `ASSERTne` and added a comment.